package.json

```diff
@@ -132,9 +132,9 @@
     "test": "mocha -r ts-node/register -r tsconfig-paths/register src/**/*.spec.ts",
     "build": "webpack --config webpack.config.js --mode=production",
     "semantic-release": "semantic-release",
-    "pack:win": "pkg -o bin/win/nexploit-cli.exe -t node14-windows-x64 .",
+    "pack:win": "copy /y node_modules\\os-service\\build\\Release\\service.node bin\\win\\service.node && pkg -o bin/win/nexploit-cli.exe -t node14-windows-x64 .",
```

Comment:
We can't use cmd commands here. It'd be better to move it to CI:

```yaml
- run: npm run pack:win
- run: npm rebuild os-service --target=v14.7.0 --arch=x64 --verbose
- run: cp node_modules/os-service/build/Release/service.node bin/win/service.node
```

Reply:
removed from package json, but dont added in CI yet